pom.xml

```diff
@@ -473,6 +473,16 @@
         <artifactId>aws-java-sdk</artifactId>
```

Comment:
Might be preferable to just remove the dependency (as opposed to excluding things).